src/pages/UserHome.jsx

```diff
@@ -41,12 +44,27 @@ const useStyles = makeStyles((theme) => ({
   noScholarshipsBox: {
     textAlign: 'center',
   },
+  progress: {
+    display: 'block',
+  },
+  loadMoreButton: {
+    margin: theme.spacing(3, 0),
+  },
 }));
 
 export default function UserHome() {
   const classes = useStyles();
+  const [scholarships, setUserHome] = useState([]);
```

Comment:
`setUserHome` is very inconsistent with the name of the state variable it sets: `scholarships`. Please rename to `setScholarships`.